### DISCUSSION TITLE
mc-docu - A multi platform tool for loading xml config backup files displaying in an electron app

### DISCUSSION THREAD

**Discussion by con-cis** [nextgenhealthcare/connect#5951] [Show and tell]
I did a side-project for the Mirth documentation tool within an electronic application and would like to share the repository with you:
https://github.com/con-cis/mc-docu
As I recall, there was a project some time ago that did something similar, but the project was discontinued. 
So far there are other useful features in the pipeline like pdf / html exports for other documentation tools and maybe a web app. i encourage you to test it and give feedback.

**Comment by jonbartels** [comment]
That is a cool project!
